lib/helpers.ts

```diff
@@ -55,7 +54,7 @@ export interface BaseSerializableEntity {
     dataset_id: string;
     level: string;
     assayName?: string;
-    primaryParents?: DataFileID[];
+    primaryParents?: synapseId[];
     synapseId?: string;
```

Comment:
might be good to also update the type of `synapseId` field

```suggestion
    synapseId?: SynapseId;
```